tests/test_sagemaker/test_server.py

```diff
@@ -0,0 +1,13 @@
+"""Test different server responses."""
+
+import moto.server as server
+
+
+def test_sagemaker_list():
```

Comment:
I know this is auto-generated by our scaffolding-script - but it's not really necessary. So we can just delete the entire file IMO

Reply:
https://github.com/getmoto/moto/pull/6454/commits/d6e0e3917a096ed1af8e0125a095a4b2dbf734fb